### ISSUE TITLE
Source Mambu : New connector request

### ISSUE THREAD

**Issue by ikanez** [airbytehq/airbyte#10030]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Source: mambu (https://api.mambu.com/)

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
Used for syncing data from the mambu banking core with our cloud analytics platform 
* How often do you want to run syncs?
Ideally stream, but batch is acceptable
* If this is an API source connector, which entities/endpoints do you need supported?
All currently available tap from singer

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Use singer.io or Meltano

## Are you willing to submit a PR?
Unlikely.

**Comment by sreeram-venkitesh** [comment]
@alafanechere Do you think this could be a good first issue to dip my toes into connector development? I have experience using Airbyte for a couple of months now and have only made a documentation contribution before. Please guide me on this :)

**Comment by alafanechere** [comment]
Yes sure @sreeram-venkitesh! Feel free to check our documentation [here](https://docs.airbyte.com/connector-development/tutorials/cdk-tutorial-python-http/getting-started). We are available on our[ Connector Development](https://discuss.airbyte.io/c/connector-development/16) forum if you need assistance.

**Comment by kristenchen** [comment]
Hi @sreeram-venkitesh, are you still working on this issue? If not, I'd like to help!

**Comment by sreeram-venkitesh** [comment]
@kristenchen Feel free to take this up 😄 I thought of doing this but I don't think I'd have the bandwidth for it now
